File: myapplication/src/main/java/com/example/myapplication/Application.java
package com.example.myapplication;


import com.facebook.drawee.backends.pipeline.Fresco;

/**
 * Created by hero on 2017/5/21.
 */

public class Application extends android.app.Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(this);
    }
}

File: gankio/src/main/java/com/example/gankio/Application.kt
package com.example.gankio

import android.app.Application
import com.facebook.cache.disk.DiskCacheConfig
import com.facebook.drawee.backends.pipeline.Fresco
import com.facebook.imagepipeline.core.ImagePipelineConfig

/**
 * Created by hero on 2017/6/17.
 */
class Application(): Application() {

    override fun onCreate() {
        super.onCreate()
        val diskCacheConfig: DiskCacheConfig = DiskCacheConfig.newBuilder(this)
                .setMaxCacheSize(Const.DISK_CACHE_MAX_SIZE)
                .setBaseDirectoryName(Const.DISK_CACHE_DIR_NAME)
                .build()
        var config: ImagePipelineConfig = ImagePipelineConfig.newBuilder(this)
                .setMainDiskCacheConfig(diskCacheConfig)
                .build()
        Fresco.initialize(this, config)
    }
}